schemas/com.snowplowanalytics.snowplow/media_player_event_ping/jsonschema/1-0-0

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Media player event fired periodicaly during main content playback, regardless of other API events that have been sent.",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "media_player_event_ping",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "label": {
```

Comment:
Does this need a time property?

Reply:
I'm not sure it does – we have the `currentTime` as a property in the media player entity as it is also relevant in other events, I don't think it would add much benefit here also considering that the ping events are sent when the video is paused as well, so they are not strictly tied to the playback time.